fullprofile.go

```diff
@@ -108,7 +109,10 @@ func GenerateProfile(gender int) *Profile {
 		Number(100, 9999),
 	)
 
-	profile.Email = strings.ToLower(profile.Name.First) + "." + strings.ToLower(profile.Name.Last) + "@example.com"
+	domain := Domain()
+	profile.Email = strings.ToLower(profile.Name.First) + "." + strings.ToLower(profile.Name.Last) + "@" + domain
```

Comment:
As I said in the other comment. I would prefer we keep this as example.com since it's a domain
whose purpose is to be used for cases like this while the other domains are actual
domains used by actual companies.

Perhaps we could add some sort of flag that could control it, by default it's
example.com but can be overridden. What do you think?